src/viewer/styles.styl

```diff
@@ -48,6 +48,22 @@
       margin-left      0
       width            100%
 
+.pho-viewer-canceled
+.pho-viewer-noviewer
+.pho-viewer-audioviewer
+.pho-viewer-textviewer
```

Comment:
Not sure about how the JSX looks like but I wonder if it wouldn't be better to have a common class `.pho-viewer` (or something else generic if it's already in use for another element) to do this.
I am not comfortable having several classes doing the same thing (my personal limit being 2)

Reply:
I agree, I really do, but I'm not looking for more enhancements on this PR. I know there's still a lot that can be improved in this code, but the PR has to stop somewhere :)

Reply:
Yes, sorry, I didn't mention it but this is cleary a non-blocking comment. :)